Block/Adminhtml/Form/Field/Accounts.php

```diff
@@ -0,0 +1,199 @@
+<?php
+namespace Kevin\Payment\Block\Adminhtml\Form\Field;
+
+use Kevin\Payment\Block\Adminhtml\Form\Field\Accounts\CountryColumn;
+use Kevin\Payment\Block\Adminhtml\Form\Field\Accounts\BankColumn;
+use Magento\Config\Block\System\Config\Form\Field\FieldArray\AbstractFieldArray;
+use Magento\Framework\DataObject;
+use Magento\Framework\Exception\LocalizedException;
+
+class Accounts extends AbstractFieldArray
```

Comment:
How about applying a better class name? Looking at what it does, it can have something like `AccountViewBuilder` which builds some kind of view.

Reply:
renamed to AccountViewBuilder